docs/user/DockerVolumeMounts.md

```diff
@@ -0,0 +1,69 @@
+# List of valid configuration variables
```

Comment:
The current title is too generic. Suggested text

# Docker Volume Mapping

Reply:
updated